front/src/modules/trainschedule/components/ManageTrainSchedule/Allowances/Allowances.tsx

```diff
@@ -4,8 +4,9 @@ import { useDispatch, useSelector } from 'react-redux';
 import { AiOutlineDash } from 'react-icons/ai';
 
 import { StandardAllowance, osrdEditoastApi, Allowance } from 'common/api/osrdEditoastApi';
-import { updateAllowances } from 'reducers/osrdconf';
-import { getAllowances, getPathfindingID } from 'reducers/osrdconf/selectors';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Reply:
Done